### DISCUSSION TITLE
Copy function

### DISCUSSION THREAD

**Discussion by vivitekk** [saber-notes/saber#946] [Ideas]
Hi,
as I was taking some notes during my physical chemistry course, I thought it would be nice to have a copy function paired with the select function. It would make taking notes of some derivations much easier as it would allow to copy part of a previous line and write only the part that has changed.

I hope it's not very hard or time consuming to implement. Appreciate your work very much!

**Comment by AraxTheCoder** [comment]
This should be implemented by #937.
It's not really copy and paste for the moment but sufficient for the use case you described.
I plan on implementing a copy cut and paste selection feature when you can also manually drag the selection to a different page.

**Reply by codeouran** [reply]
I was also wondering about being able to use the drag the selected text to another page. I have been running into that issue today. I want to drag selected notes to a new page and can't

**Comment by guretalde** [comment]
Is there any update on the possible implementation of cut and paste?

Thank you for all the effort in this app!!!

**Comment by klausrauchelektronik** [comment]
Would be great to have the copy/paste function, also to move something to another page!

**Reply by nilomind-code** [reply]
Please add cut and paste, also between pages/documents.

**Reply by klausrauchelektronik** [reply]
Is there a roadmap, when and if, the function should be implemented?

Great app, I like to use it every day!

**Reply by Synt4x3rror** [reply]
> Please add cut and paste, also between pages/documents.

I would also like it vert much if you could copy between pages. Really love the app. Just recently began using it and it checks most of the boxes of what I would want it to do. Thank you for the good job done so far :)
